### PR TITLE
Add gin log formatter

### PR DESCRIPTION
**Pull request by babygoat**

### CODE REVIEW

**Comment by babygoat** [comment]
@nickhsine  As the go-logformatter is not officially released, we could directory add the go-logformatter module name(github.com/twreporter/go-logformatter) in go.mod for local integration test

```
import (
...
github.com/twreporter/go-logformatter v0.0.0
...
)
```

and use `replace` directive to import the package(module) from local.

```
replace github.com/twreporter/go-logformatter v0.0.0 => Local module path(Note: should start with ./ or ../)
```

**Comment by babygoat** [comment]
@nickhsine

File: stackdriver.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+	httpRequest struct {
+		RequestMethod string `json:"requestMethod"`
+		RequestUrl    string `json:"requestUrl"`
+		Status        int    `json:"status"`
+		UserAgent     string `json:"userAgent"`
+		RemoteIp      string `json:"remoteIp"`
+		Latency       string `json:"latency"`
+		Protocol      string `json:"protocol"`
+		ResponseSize  string `json:"responseSize"`
+	}
```
Comment (nickhsine) [inline stackdriver.go]: Use official pkgs?
https://godoc.org/google.golang.org/genproto/googleapis/logging/v2#LogEntry
https://godoc.org/google.golang.org/genproto/googleapis/logging/type#HttpRequest

File: stackdriver.go [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+	Default Severity = iota
+	Debug
+	Info
+	Notice
+	Warning
+	Error
+	Critical
+	Alert
+	Emergency
+)
```
Comment (nickhsine) [inline stackdriver.go]: Use official pkg?
https://godoc.org/google.golang.org/genproto/googleapis/logging/type#LogSeverity

**Comment by babygoat** [comment]
Updated.

**Comment by nickhsine** [comment]
LGTM

**Pull request merged**